src/app/public/modules/popover/popover.component.ts

```diff
@@ -166,7 +168,10 @@ export class SkyPopoverComponent implements OnInit, OnDestroy {
 
     // Let the styles render before gauging the dimensions.
     this.windowRef.getWindow().setTimeout(() => {
-      if (this.adapterService.isPopoverLargerThanParent(this.popoverContainer)) {
+      if (
+        this.adapterService.isPopoverLargerThanParent(this.popoverContainer)
+        && !this.containOverflow
```

Comment:
Booleans are calculated faster than methods. I'd put the boolean argument first so that if it fails, the height calculation isn't even attempted (could save on performance).

Reply:
Done.